backend/src/main/java/gov/cdc/usds/simplereport/api/converter/ConditionAgnosticConvertToPatientProps.java

```diff
@@ -0,0 +1,14 @@
+package gov.cdc.usds.simplereport.api.converter;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Builder
+@Getter
+public class ConditionAgnosticConvertToPatientProps {
```

Comment:
creating these classes to use in a similar way to [the existing versions for `TestResultUpload`.](https://github.com/CDCgov/prime-simplereport/blob/main/backend/src/main/java/gov/cdc/usds/simplereport/api/converter/ConvertToPatientProps.java). Thought that since the new CA schema was missing half the values that it'd be better to make new prop objects rather than add null checks for the existing one for half the fields.